docs/linting/troubleshooting/FORMATTING.md

```diff
@@ -22,6 +22,11 @@ That generally makes them run orders of magnitude slower.
 Additionally, modern formatters such as Prettier are architected in a way that applies formatting to all code regardless of original formatting.
 Linters typically run on a rule-by-rule basis, typically resulting in many edge cases and missed coverage in formatting.
 
+### Suggested Usage
+
+We recommend using [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier) to disable formatting rules in your ESLint configuration.
+You can then configure your formatter separately from ESLint.
```

Comment:
Err, this isn't showing up in https://deploy-preview-5427--typescript-eslint.netlify.app/docs/linting/troubleshooting/formatting/:

<img width="882" alt="Screenshot of the deployed page showing no Suggested Usage section before ESLint Core and Formatting" src="https://user-images.githubusercontent.com/3335181/183199584-00d61928-1642-44fc-aee4-efe06e2aba76.png">

...something off with Netlify build caches...?

Reply:
Well, regardless of whether this is fixed by c25e1dc, it's external to this PR. 🤷